TLM/TLM/Custom/PathFinding/CustomPathManager.cs

```diff
@@ -28,22 +25,70 @@ public class CustomPathManager : PathManager {
 
         private CustomPathFind[] _replacementPathFinds;
 
-        public static CustomPathManager _instance;
+        public static CustomPathManager _instance => PathManager.instance as CustomPathManager;
+
+        private PathManager stockPathManager_;
+
+        private static FastList<ISimulationManager> GetSimulationManagers() =>
+            typeof(SimulationManager)
+            .GetField("m_managers", BindingFlags.Static | BindingFlags.NonPublic)
+            ?.GetValue(null)
+            as FastList<ISimulationManager>
+            ?? throw new Exception("could not get SimulationManager.m_managers");
+
+        private static FieldInfo PathManagerInstance =>
+            typeof(Singleton<PathManager>)
+            .GetField(
+            "sInstance",
+            BindingFlags.Static | BindingFlags.NonPublic) ??
+            throw new Exception("pathManagerInstance is null");
 
 #if QUEUEDSTATS
-        public static uint TotalQueuedPathFinds {
-            get; private set;
-        }
+        public static uint TotalQueuedPathFinds { get; private set; }
 #endif
 
-        public static bool InitDone {
-            get; private set;
+        public static void OnLevelLoaded() {
+            try {
+                Log.Info("CustomPathManager.OnLevelLoaded() called.");
+                PathManager.instance.gameObject.AddComponent<CustomPathManager>();
+                Log._Debug("Added CustomPathManager to gameObject List");
```

Comment:
This message carries no value. It works? Delete the debug log.

Reply:
>  Does it work? 

I don't understand.
I just copied it from LoadingExtension to here. But I can delete it.

Reply:
It looks like it was used by the person who wrote this code to debug timing when it was created, or something like that. Now it just creates noise in logs and does not report any success or error.